internal/server/iigointernal/judiciary.go

```diff
@@ -253,7 +391,7 @@ func defaultInitLocalHistoryCache(depth int) map[int][]shared.Accountability {
 	return returnMap
 }
 
-func generateEmptyCommunicationsMap(allTeamIds []shared.ClientID) map[shared.ClientID][]map[shared.CommunicationFieldName]shared.CommunicationContent {
+func generateEmptyCommunicationsMap(allTeamIds [6]shared.ClientID) map[shared.ClientID][]map[shared.CommunicationFieldName]shared.CommunicationContent {
```

Comment:
Having the 6 hard coded is to do with the number of islands correct. Should this value not be taken from the gamestate or config so that we can run games with a varying number of islands

Reply:
Yeah good catch, I've set it to feed off the length of the shared.TeamIds, I though it would be the best length for the array initialiser.